ddtrace/appsec/processor.py

```diff
@@ -210,9 +217,51 @@ def __attrs_post_init__(self):
         # we always need the response headers
         self._mark_needed(_Addresses.SERVER_RESPONSE_HEADERS_NO_COOKIES)
 
+    def update_rules(self, new_rules):
```

Comment:
note that this will be a public API if released, is that intentional?

Reply:
Probably not. Until I can ask @avara1986 tomorrow I will make it private.